### ISSUE TITLE
Sitemap Generation Error

### ISSUE THREAD

**Issue by mortscode** [nystudio107/craft-seomatic#985]
### Describe the bug
Added a page to my site, trying to generate the sitemap and I'm getting the following error:
```
2021-09-26 00:41:28 [-][-][-][info][nystudio107\seomatic\Seomatic::init] SEOmatic plugin loaded
2021-09-26 00:41:28 [-][-][-][info][dolphiq\redirect\RedirectPlugin::init] dolphiq/redirect Plugin plugin loaded
2021-09-26 00:41:28 [-][-][-][info][mortscode\feedback\Feedback::init] Feedback plugin loaded
2021-09-26 00:41:29 [-][-][-][info][nystudio107\seomatic\services\Sitemaps::invalidateSitemapCache] Sitemap cache cleared: cookbook
2021-09-26 00:41:29 [-][-][-][info][craft\queue\QueueLogBehavior::beforeExec]  [378609] Generating cookbook sitemap (attempt: 1, pid: 37) - Started
2021-09-26 00:41:29 [-][-][-][error][nystudio107\seomatic\helpers\UrlHelper::absoluteUrlWithProtocol] Getting unknown property: yii\console\Response::statusCode
2021-09-26 00:41:29 [-][-][-][error][craft\queue\QueueLogBehavior::afterError]  [378609] Generating cookbook sitemap (attempt: 1, pid: 37) - Error (time: 0.030s): Getting unknown property: yii\console\Response::statusCode
```

### To reproduce
Steps to reproduce the behaviour:
1. Log into servd backend
2. In the Commands section of Production, run:
`seomatic/sitemap/generate --handle=[new page handle]`

### Expected behaviour
Generate sitemap

### Versions
- Plugin version: 3.4.12
- Craft version: 3.7.13

**Comment by mortscode** [comment]
I went back to version 3.4.10 and everything seems to be working now.

**Comment by khalwat** [comment]
Addressed in: https://github.com/nystudio107/craft-seomatic/commit/15e9cc5159d728ec929f1bb337bdc74826069aa4

Released as 3.4.13 -> https://github.com/nystudio107/craft-seomatic/releases/tag/3.4.13
